test/container/builder/run-kojid.sh

```diff
@@ -1,12 +1,8 @@
 #!/bin/bash
 set -ux
 
-if ls /share/rpms/*.rpm 1> /dev/null 2>&1; then
-  echo "Using RPMs"
-  rm /usr/lib/koji-builder-plugins/osbuild.py
-  rpm -i /share/rpms/koji-osbuild-?-0.*.rpm \
-         /share/rpms/koji-osbuild-builder-*.rpm
-fi
+rpm -i /share/rpms/koji-osbuild-?-0.*.rpm \
+       /share/rpms/koji-osbuild-builder-*.rpm
```

Comment:
What if we kept the `if`, but removed the `rm`. Then add an `elif ls` to see if `/share/plugins/builder/osbuild.py` exists and if so copy that over.